general/administration/installing.md

```diff
@@ -220,13 +217,13 @@ Generic amd64 Linux builds in TAR archive format are available [here](https://je
 ### Installation Process
 
 Create a directory in /opt for jellyfin and its files, and enter that directory.
-    
+
 ```bash
-sudo mkdir /opt/jellyfin 
+sudo mkdir /opt/jellyfin
 cd /opt/jellyfin
 ```
 
-Download the latest generic linux build from the [Jellyfin release page](https://github.com/jellyfin/jellyfin/releases). The generic linux build ends with "linux-amd64.tar.gz". The rest of these instructions assume version 10.4.3 is being installed (i.e. jellyfin_10.4.3_linux-amd64.tar.gz). Download the generic build, then extract the archive:
+Download the latest generic linux build from the [release page](https://github.com/jellyfin/jellyfin/releases). The generic linux build ends with "linux-amd64.tar.gz". The rest of these instructions assume version 10.4.3 is being installed (i.e. jellyfin_10.4.3_linux-amd64.tar.gz). Download the generic build, then extract the archive:
```

Comment:
```suggestion
Download the latest generic Linux build from the [release page](https://github.com/jellyfin/jellyfin/releases). The generic Linux build ends with "linux-amd64.tar.gz". The rest of these instructions assume version 10.4.3 is being installed (i.e. `jellyfin_10.4.3_linux-amd64.tar.gz`). Download the generic build, then extract the archive:
```

Reply:
Linux+gnu! Haha. Not really, RMS will survive. Besides, at this point, it outta be Linux+gnu+systems >.>